backend/people/migrations/0019_fill_more_seminars.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2020-10-22 19:47
+from __future__ import unicode_literals
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    Seminar = apps.get_model('events', 'Seminar')
+    WSS = apps.get_model('WSS', 'WSS')
+    Speaker = apps.get_model('people', 'Speaker')
+
+    wss = WSS.objects.get(year=2020)
+
+    Seminar.objects.create(
+        wss=wss,
+        title='Appraisal of the Network of caches',
+        abstract='The growth of the Internet in both size and usage is beyond expectations. New ideas and techniques have emerged to facilitate end-users to receive seamless services. One of the significant successful solutions is caching multiple data copies in the network as a Content Delivery Network service. Content Delivery Network is an application service that redirects the clients\' requests toward a closer copy of data. A network of caches is a finer granularity service that provides a caching system at the network level. Some of the service\'s benefits are reduction in the network load, less download latency, and a means to ease mobile communications. However, searching the memory, especially when data is not available, is a waste of bandwidth and router\'s resources. Therefore, enhancing the performance of the network of caches is a challenge to overcome. \nIn this talk, we first introduce the Named Data Networking platform to enable caching capability at the network layer. Then we investigate the challenges that need to be addressed to provide caching at this level. A review of solutions will be discussed along with some experimental results and our approaches to make the network closer to the reality with enhanced performance.',
+        speaker=Speaker.objects.filter(name='Mostafa Rezazad').last()
+    )
```

Comment:
Please set the `polymorphic_ctype` for these objects.

Reply:
check migration number 11
do I need to set this for them too?